compose/compose-file/index.md

```diff
@@ -1610,18 +1610,24 @@ services:
     ports:
       - "80:80"
 
+    networks:
+      webnet:
+
+    volumes:
+      - type: volume
+        source: mydata
+        target: /data
+        volume:
+          nocopy: true
+      - type: bind
+        source: ./static
+        target: /opt/app/static
+        
 networks:
   webnet:
 
 volumes:
-  - type: volume
-    source: mydata
-    target: /data
-    volume:
-      nocopy: true
-  - type: bind
-    source: ./static
-    target: /opt/app/static
+  mydata:
```

Comment:
Don't you also want to give it access to the bind mount?

Reply:
@mstanleyjones , I think it has access as is. See:

* https://github.com/docker/compose/issues/5092#issuecomment-321365861
* https://docs.docker.com/compose/compose-file/#volumes

Reply:
Ahh I see, you don't have to explicitly grant the service access to the bind mount.